Pcap++/src/PcapLiveDevice.cpp

```diff
@@ -261,6 +261,17 @@ pcap_t* PcapLiveDevice::doOpen(const DeviceConfiguration& config)
 	}
 #endif
 
+#ifdef HAS_DIRECTION_SET
+    ret = pcap_setdirection(pcap, config.direction);

```

Comment:
is the default value `PCAP_D_INOUT`? meaning, is not using `pcap_setdirection` similar to use it with `PCAP_D_INOUT`? I want to make sure everything is backward compatible

Reply:
Yes, it is.

Reply:
Awesome thanks for verifying that